src/renderer/features/governance/model/unlock.ts

```diff
@@ -21,6 +21,7 @@ const txSaved = createEvent();
 
 const $step = restore<Step>(stepChanged, Step.NONE);
 const $claimSchedule = createStore<UnlockChunk[]>([]).reset(walletModel.$activeWallet);
+const $totalUnlock = createStore<BN | null>(null);
```

Comment:
what if use `BN_ZERO` as default? It will helps to skip `return unlockable.isZero() ? null : unlockable;` and use `!unlockable.isZero()` instead of `!!totalUnlock` in `UnlockInfo.tsx`

Just suggestion